src/views/clusteroverview/MonitoringTab/migrations-card/components/MigrationsTable/MigrationsRow.tsx

```diff
@@ -0,0 +1,124 @@
+  TableData,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { Tooltip } from '@patternfly/react-core';
+
+import { iconMapper, vmimStatuses } from './utils/statuses';
+import { MigrationTableDataLayout } from './utils/utils';
+import MigrationActionsDropdown from './MigrationActionsDropdown';
+
+const MigrationsRow: React.FC<RowProps<MigrationTableDataLayout>> = ({ obj, activeColumnIDs }) => {
+  const { t } = useKubevirtTranslation();
+  const StatusIcon = iconMapper[obj?.vmim?.status?.phase];
+  return (
+    <>
+      <TableData id="vm-name" activeColumnIDs={activeColumnIDs}>
+        <ResourceLink
+          groupVersionKind={VirtualMachineModelGroupVersionKind}
+          name={obj?.vmiObj?.metadata?.name}
+          namespace={obj?.vmiObj?.metadata?.namespace}
+        />
+      </TableData>
+      <TableData id="status" activeColumnIDs={activeColumnIDs}>
+        <Tooltip
+          content={`${obj?.vmim?.status?.phase} ${
+            obj?.vmiObj?.status?.migrationState?.endTimestamp || ''
+          }`}
+          hidden={
+            obj?.vmim?.status?.phase !== vmimStatuses.Failed &&
+            obj?.vmim?.status?.phase !== vmimStatuses.Succeeded
+          }
+        >
+          <GenericStatus title={obj?.vmim?.status?.phase} Icon={StatusIcon} />
+        </Tooltip>
+      </TableData>
+      <TableData id="source" activeColumnIDs={activeColumnIDs}>
+        {obj?.vmiObj?.status?.migrationState?.sourceNode ? (
+          <ResourceLink
+            groupVersionKind={modelToGroupVersionKind(NodeModel)}
+            name={obj?.vmiObj?.status?.migrationState?.sourceNode}
+          />
+        ) : (
+          NO_DATA_DASH
+        )}
+      </TableData>
+      <TableData id="target" activeColumnIDs={activeColumnIDs}>
+        {obj?.vmiObj?.status?.migrationState?.targetNode ? (
+          <ResourceLink
+            groupVersionKind={modelToGroupVersionKind(NodeModel)}
+            name={obj?.vmiObj?.status?.migrationState?.targetNode}
+          />
+        ) : (
+          NO_DATA_DASH
+        )}
+      </TableData>
+      <TableData id="migration-policy" activeColumnIDs={activeColumnIDs}>
+        {obj?.vmiObj?.status?.migrationState?.migrationPolicyName ? (
+          <Tooltip
+            content={
+              <>
+                <div>
+                  {t('Bandwidth per migration')}:{' '}
+                  {readableSizeUnit(obj?.mpObj?.spec?.bandwidthPerMigration)}
+                </div>
+                <div>
+                  {t('Auto converge')}: {obj?.mpObj?.spec?.allowAutoConverge ? t('Yes') : t('No')}
+                </div>
+                <div>
+                  {t('Post copy')}: {obj?.mpObj?.spec?.allowPostCopy ? t('Yes') : t('No')}
+                </div>
+                <div>
+                  {t('Completion timeout')}: {obj?.mpObj?.spec?.completionTimeoutPerGiB}
+                </div>
+              </>
+            }
+          >
+            <ResourceLink
+              groupVersionKind={{
```

Comment:
can we use modelToGroupVersionKind or something similar here?

Reply:
we dont have migrationpolicymodel?

Reply:
I've send a PR to add this model to kubevirt-api: https://github.com/kubevirt-ui/kubevirt-api/pull/23
but we need to build a new release for it